fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -273,11 +275,21 @@ def refresh_trainable(self) -> None:
         self._setup_backward_hooks()
 
     def reduce(self) -> None:
```

Comment:
Do we need to update any docs/change log since we are supporting this feature now?

Reply:
the docs should be autogenerated from the docstring, but I'll add that to the changelog, good point